### PR TITLE
Edits for issue #42

### LINKED ISSUE #42: Register as a Validator: broken legacy link to `additional helpers`
The page has a broken link to additional helpers entry for `jq`:

>This guide also assumes that the command-line JSON processor jq is available - see [additional helpers](https://docs.autonity.org/developer/#additional-helpers).

The content was on the original 'How to' home page before restructuring (see: [content/docs/howto/_index.md](https://github.com/autonity/docs.autonity.org/commit/09d71077ce0344cd32336b5992e008d02ae62e27#diff-b340593bf455020812305a1e6e0c8f0ae669af2894f091660d6a5f0e3d535566)) :

>### Additional Helpers
>
>In some of the procedures, we will perform commands that will use [`curl`](https://curl.haxx.se/download.html). You may already have `curl` installed on your system as it comes with many OS distributions.
>
>A good helper for working with JSON response objects returned by RPC is [`jq`](https://stedolan.github.io/jq/download/) (optional).

**Register as a validator** doesn't use `jq` in the examples so this link is legacy.

To resolve:

- Remove broken, legacy link from **Register as a validator** 
- Consider adding the helpers, perhaps in an alert box type Note?) to [Autonity Contract Interface](https://docs.autonity.org/reference/api/aut/) page. Rationale: the RPC examples make use of `curl` and `jq` could also be used.

### PR DESCRIPTION
**Pull request by jay-cousins**
Minor PR to address issue #42 

Edits made:

- Remove the legacy link from Register as a Validator
- Add a Card to the Interfaces home to add `curl` and `jq` additional helpers if using RPC to read state.

closes #42

### CODE REVIEW

**Comment by jay-cousins** [comment]
Any reviewers for this please?

File: content/docs/reference/api/_index.md [hunk 1]
```diff
@@ -7,3 +7,11 @@ description: >
   Autonity Contract Interfaces and JSON-RPC APIs provided by the Autonity Go Client
 ---
 
+{{< card header="**Helper libraries**" title="Useful helpers for working with RPC" >}}
+Interface usage and examples are shown using the Autonity Utility Tool ([aut](/account-holders/setup-aut/)). RPC is also used for calls reading state.
```
Comment (lorenzo-dev1) [inline content/docs/reference/api/_index.md:11]:
```suggestion
Interface usage and examples are shown using the Autonity Utility Tool ([aut](/account-holders/setup-aut/)). RPC through curl are also used for calls reading state.
```

Comment (lorenzo-dev1) [inline content/docs/reference/api/_index.md]:
```suggestion
Interface usage and examples are shown using the Autonity Utility Tool ([aut](/account-holders/setup-aut/)).
```

File: content/docs/reference/api/_index.md [hunk 2]
```diff
@@ -7,3 +7,11 @@ description: >
...
+
+RPC examples make use of [`curl` <i class='fas fa-external-link-alt'></i>](https://curl.haxx.se/download.html). You may already have `curl` installed on your system as it comes with many OS distributions.
```
Comment (lorenzo-dev1) [inline content/docs/reference/api/_index.md]:
```suggestion
Remote Procedure Call (RPC) that read state can also be done using [`curl` <i class='fas fa-external-link-alt'></i>](https://curl.haxx.se/download.html). You may already have `curl` installed on your system as it comes with many OS distributions.
```

**Review by lorenzo-dev1 — APPROVED** [review]
@cmjc LGTM. Left some minor rewordings but feel free to discard if you think it is better as you originally wrote it.

**Pull request merged**